sphinx_external_toc/parsing.py

```diff
@@ -29,26 +29,32 @@
 class FileFormat:
     """Mapping of keys for subtrees and items, dependant on depth in the ToC."""
 
-    subtrees_keys: Sequence[str] = attr.ib(())
-    items_keys: Sequence[str] = attr.ib(())
-    default_subtrees_key: str = attr.ib(DEFAULT_SUBTREES_KEY)
-    default_items_key: str = attr.ib(DEFAULT_ITEMS_KEY)
     toc_defaults: Dict[str, Any] = attr.ib(factory=dict)
+    subtrees_keys: Sequence[str] = attr.ib(default=())
+    items_keys: Sequence[str] = attr.ib(default=())
+    default_subtrees_key: str = attr.ib(default=DEFAULT_SUBTREES_KEY)
+    default_items_key: str = attr.ib(default=DEFAULT_ITEMS_KEY)
 
     def get_subtrees_key(self, depth: int) -> str:
-        """Get the subtrees key name for this depth in the ToC
+        """Get the subtrees key name for this depth in the ToC.
 
         :param depth: recursive depth (starts at 0)
+        :type depth: str
```

Comment:
`type`/`rtype` are not necessary in the docstring, since they are provided by the type annotations

Reply:
Gotcha